style.css

```diff
@@ -360,22 +360,72 @@ h4 {
   color: gray;
 }
 
-.modal1 {
-  background-color: #fff;
+.modal-overlay {
   position: fixed;
+  background-color: #c1c7d0;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  display: none;
```

Comment:
Kindly ensure that your project sticks as much as possible to the Figma template.  Kindly fix the following issues.
- The modal window should show all the information without the need to scroll down. :pray: :pray: 
- The modal window's header section is different from the Figma template. :eyes:  
- The modal window's image is too big compared to the Figma template. :eyes: 
- The buttons and the text are organized in a different way compared to the Figma template.
- Kindly add a bottom margin to the modal window to make it similar to the Figma template. :+1: :smiley: :smiley: 
Your design
![image](https://user-images.githubusercontent.com/98363075/197089893-85fa3fac-a043-4b64-8e39-5d8aab512bb1.png)
![image](https://user-images.githubusercontent.com/98363075/197090052-e2342565-8a57-4343-9ff5-8955687e70da.png)


Figma design
![image](https://user-images.githubusercontent.com/98363075/197090109-e314f2ae-8b3c-4aa0-bb65-f1d6c98d93c8.png)